File: MPI/mpicheck/tests/functions/2.0/MPI_File_get_position.f90

        
        program main
        use mpi
        
        INTEGER var_0
       INTEGER(KIND=MPI_OFFSET_KIND) var_1
       INTEGER var_2
        call mpi_file_get_position(var_0, var_1, var_2)
       call pmpi_file_get_position(var_0, var_1, var_2)
        end program main
    

File: MPI/mpicheck/tests/functions/2.0/MPI_File_get_position.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    MPI_File var_0;
    MPI_Offset *var_1;
    int ret;
    /* calls */
    ret = MPI_File_get_position(var_0, var_1);
    ret = PMPI_File_get_position(var_0, var_1);
    return 0;
}
